util/fastrand/main_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fastrand
+
+import (
+	"os"
+	"testing"
+
+	"github.com/pingcap/tidb/util/testbridge"
+)
+
+func TestMain(m *testing.M) {
+	testbridge.WorkaroundGoCheckFlags()
+	os.Exit(m.Run())
```

Comment:
```
func TestMain(m *testing.M) {
	testbridge.WorkaroundGoCheckFlags()
	goleak.VerifyTestMain(m)
}
```

Take a closer look at #26082. I suggest you introduce goleak to prevent potential goroutine leaking.

Reply:
I have fixed this problem, however I cannot find `goleak` in #26082, found in #26252 instead.

Reply:
Aha, I should update the example. Best practice updated.